airlock_processor/StatusChangedQueueTrigger/__init__.py

```diff
@@ -104,78 +102,130 @@ def is_require_data_copy(new_status: str):
+    error_message = f"Missing copy destination storage account definition for status '{new_status}' and request type '{request_type}'."
+    logging.error(error_message)
+    raise Exception(error_message)
 
 
-def set_output_event_to_report_failure(outputEvent, request_properties, failure_reason, request_files):
+def set_output_event_to_report_failure(stepResultEvent, request_properties, failure_reason, request_files):
     logging.exception(f"Failed processing Airlock request with ID: '{request_properties.request_id}', changing request status to '{constants.STAGE_FAILED}'.")
-    outputEvent.set(
+    stepResultEvent.set(
         func.EventGridOutputEvent(
             id=str(uuid.uuid4()),
-            data={"completed_step": request_properties.status, "new_status": constants.STAGE_FAILED, "request_id": request_properties.request_id, "request_files": request_files, "error_message": failure_reason},
+            data={"completed_step": request_properties.new_status, "new_status": constants.STAGE_FAILED, "request_id": request_properties.request_id, "request_files": request_files, "error_message": failure_reason},
             subject=request_properties.request_id,
             event_type="Airlock.StepResult",
             event_time=datetime.datetime.utcnow(),
             data_version=constants.STEP_RESULT_EVENT_DATA_VERSION))
 
 
-def set_output_event_to_report_request_files(outputEvent, request_properties, request_files):
-    outputEvent.set(
+def set_output_event_to_report_request_files(stepResultEvent, request_properties, request_files):
+    logging.info(f'Sending file enumeration result for request with ID: {request_properties.request_id} result: {request_files}')
+    stepResultEvent.set(
         func.EventGridOutputEvent(
             id=str(uuid.uuid4()),
-            data={"completed_step": request_properties.status, "request_id": request_properties.request_id, "request_files": request_files},
+            data={"completed_step": request_properties.new_status, "request_id": request_properties.request_id, "request_files": request_files},
             subject=request_properties.request_id,
             event_type="Airlock.StepResult",
             event_time=datetime.datetime.utcnow(),
             data_version=constants.STEP_RESULT_EVENT_DATA_VERSION))
 
 
-def get_request_files(request_properties):
-    containers_metadata = get_source_dest_for_copy(request_properties.status, request_properties.type, request_properties.workspace_id)
-    storage_account_name = containers_metadata.source_account_name
+def set_output_event_to_trigger_container_deletion(toDeleteEvent, request_properties, container_url):
+    logging.info(f'Sending container deletion event for request ID: {request_properties.request_id}. container URL: {container_url}')
+    toDeleteEvent.set(
+        func.EventGridOutputEvent(
+            id=str(uuid.uuid4()),
+            data={"blob_to_delete": container_url},
```

Comment:
is it container url or blob url?
if we delete a blob or container we need to align all args naming

Reply:
we're passing the container URL here.
to clear things up:
before this PR, toDeleteTrigger, supported blob deletion only, it was done by providing the blob URL. e.g.:
`https://account.blob.core.windows.net/container/blob`

my changes made toDeleteTrigger support container deletion as well, it is done by providing the container URL. e.g.:
`https://account.blob.core.windows.net/container/`